addons/l10n_es/data/account_tax_data.xml

```diff
@@ -1,300 +1,10 @@
-        <field name="name">mod303[33]</field>
-        <field name="applicability">taxes</field>
-        <field name="country_id" ref="base.es"/>
-    </record>
-    <record id="mod_303_34" model="account.account.tag">
-        <field name="name">mod303[34]</field>
-        <field name="applicability">taxes</field>
-        <field name="country_id" ref="base.es"/>
-    </record>
-    <record id="mod_303_35" model="account.account.tag">
-        <field name="name">mod303[35]</field>
-        <field name="applicability">taxes</field>
-        <field name="country_id" ref="base.es"/>
-    </record>
-    <record id="mod_303_36" model="account.account.tag">
-        <field name="name">mod303[36]</field>
-        <field name="applicability">taxes</field>
-        <field name="country_id" ref="base.es"/>
-    </record>
-    <record id="mod_303_37" model="account.account.tag">
-        <field name="name">mod303[37]</field>
-        <field name="applicability">taxes</field>
-        <field name="country_id" ref="base.es"/>
-    </record>
-    <record id="mod_303_38" model="account.account.tag">
-        <field name="name">mod303[38]</field>
-        <field name="applicability">taxes</field>
-        <field name="country_id" ref="base.es"/>
-    </record>
-    <record id="mod_303_39" model="account.account.tag">
-        <field name="name">mod303[39]</field>
-        <field name="applicability">taxes</field>
-        <field name="country_id" ref="base.es"/>
-    </record>
-    <record id="mod_303_40" model="account.account.tag">
-        <field name="name">mod303[40]</field>
-        <field name="applicability">taxes</field>
-        <field name="country_id" ref="base.es"/>
-    </record>
-    <record id="mod_303_41" model="account.account.tag">
-        <field name="name">mod303[41]</field>
-        <field name="applicability">taxes</field>
-        <field name="country_id" ref="base.es"/>
-    </record>
-    <record id="mod_303_42" model="account.account.tag">
-        <field name="name">mod303[42]</field>
-        <field name="applicability">taxes</field>
-        <field name="country_id" ref="base.es"/>
-    </record>
-    <record id="mod_303_59" model="account.account.tag">
-        <field name="name">mod303[59]</field>
-        <field name="applicability">taxes</field>
-        <field name="country_id" ref="base.es"/>
-    </record>
-    <record id="mod_303_60" model="account.account.tag">
-        <field name="name">mod303[60]</field>
-        <field name="applicability">taxes</field>
-        <field name="country_id" ref="base.es"/>
-    </record>
     <record id="mod_303_61" model="account.account.tag">
         <!--Not used anymore since Q3 2021; replaced by grids 120, 122, 123 and 124-->
         <field name="name">mod303[61]</field>
         <field name="applicability">taxes</field>
         <field name="active" eval="False"/>
         <field name="country_id" ref="base.es"/>
     </record>
```

Comment:
I think this refactor would also be a good opportunity to remove this account tag once and for all :smile: